src/styles/index.scss

```diff
@@ -1,3 +1,120 @@
+$main: #060b35;
+$secondary: #616070;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  flex-direction: column;
+  width: 200px;
+  border: solid 1px #f3f3f3;
+  display: flex;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+  padding: 32px 16px 16px;
+}
+
+.card__picture {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__name {
+  margin: 0 0 4px;
+  color: $main;
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.card__code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: $secondary;
+}
+
+.card__reviews {
+  display: flex;
+  margin-bottom: 24px;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
there should be 1 file for every BEM block - according to [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) point 11 - please create 1 file for stars and 1 for card